frontend/pages/DashboardPage/cards/MDM/MDMStatusTableConfig.tsx

```diff
@@ -56,10 +56,7 @@ export const generateStatusTableHeaders = (teamId?: number): IDataColumn[] => [
       !MDM_STATUS_TOOLTIP[status] ? (
         <TextCell value={status} />
       ) : (
-        <TooltipWrapper
-          position="top-start"
```

Comment:
Same question for product as [below](https://github.com/fleetdm/fleet/pull/19002/files#r1601737620)